### PR TITLE
Update Content Build Separation design doc

### PR DESCRIPTION
**Pull request by bkjohnson**
This is to account for the decision to use two s3 buckets.

Related to https://github.com/department-of-veterans-affairs/va.gov-team/issues/10001

You can read the full document [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/content-build-design-doc-revision/platform/engineering/design-docs/2020-04-09-separate-content-build.md)

### CODE REVIEW

File: platform/engineering/design-docs/2020-04-09-separate-content-build.md [hunk 1]
```diff
@@ -168,8 +168,7 @@ The current front end build will be split up into two distinct builds:
     - Output: JavaScript and CSS bundles
     - This essentially maps to the current full deploy minus the content
 
-The output of both these builds will be deployed to a single S3 bucket with
-safeguards in place to ensure they don't override each other.
+The output of the content build will be deployed to one s3 bucket, and the output of the application build will be deployed to a different one. The separate bucket for the content build output will be new.
```
Comment (cvalarida) [inline platform/engineering/design-docs/2020-04-09-separate-content-build.md:171]: It sounds like the application build will use the current S3 bucket used to hold both applications and content. Mind clarifying that?

Comment (cvalarida) [inline platform/engineering/design-docs/2020-04-09-separate-content-build.md:171]:
```suggestion
The output of the content build will be deployed to one S3 bucket, and the output of the application build will be deployed to a different one. The separate bucket for the content build output will be new.
```

File: platform/engineering/design-docs/2020-04-09-separate-content-build.md [hunk 2]
```diff
@@ -208,18 +207,10 @@ which corresponds to the URLs on VA.gov. For example, `/education/index.html` is
 the file that's served at www.va.gov/education/.
 
 The output of the application build **currently** live in the `/generated/`
-directory. To support the transition, the **new** application build will put its
-assets in the `/applications/` directory.
+directory. By using different buckets for the application build output and content build output, the `/generated` directory can be removed.
```
Comment (cvalarida) [inline platform/engineering/design-docs/2020-04-09-separate-content-build.md:210]: How will requests to the content be routed to the content bucket? How will requests for application assets be routed to the application bucket?

File: platform/engineering/design-docs/2020-04-09-separate-content-build.md [hunk 3]
```diff
@@ -232,8 +223,6 @@ To ensure that one build won't override files in the other, we'll have to
   tests can run a browser and access the applications
 - **Important note:** These HTML files will not be served in production; the
   landing pages for the applications come from the content build
-  - To ensure these pages aren't made in production, the deploy script will copy
-    only the files in `/applications/`
```
Comment (cvalarida) [inline platform/engineering/design-docs/2020-04-09-separate-content-build.md:236]: I think this part is still applicable, but less important with the separated buckets. We still don't want to upload the fake landing pages to S3, I think.

File: platform/engineering/design-docs/2020-04-09-separate-content-build.md [hunk 4]
```diff
@@ -496,3 +474,4 @@ There are a few challenges that multiple buckets present.
 Date | Revisions Made | Author
 -----|----------------|--------
 Apr 9, 2020 | Initial draft | Christopher Valarida
+Jun 11, 2020 | Change the plan to use multiple s3 buckets | Brooks Johnson
```
Comment (cvalarida) [inline platform/engineering/design-docs/2020-04-09-separate-content-build.md:477]:
```suggestion
Jun 11, 2020 | Change the plan to use multiple S3 buckets | Brooks Johnson
```

**Comment by bkjohnson** [comment]
@cvalarida I've made some clarifications in response to your feedback.

File: platform/engineering/design-docs/2020-04-09-separate-content-build.md [hunk 5]
```diff
@@ -168,8 +168,8 @@ The current front end build will be split up into two distinct builds:
     - Output: JavaScript and CSS bundles
     - This essentially maps to the current full deploy minus the content
 
-The output of both these builds will be deployed to a single S3 bucket with
-safeguards in place to ensure they don't override each other.
+There will be two S3 buckets. One will hold the output of the application build, and the other will hold the output of the content build.
```
Comment (rianfowler) [inline platform/engineering/design-docs/2020-04-09-separate-content-build.md:171]:
```suggestion
There will be two S3 buckets. One will hold the output of the application build (including .js and .css), and the other will hold the output of the content build (including .html and .pdf).
```

**Review by rianfowler — APPROVED** [review]

File: platform/engineering/design-docs/2020-04-09-separate-content-build.md [hunk 6]
```diff
@@ -168,8 +168,8 @@ The current front end build will be split up into two distinct builds:
     - Output: JavaScript and CSS bundles
     - This essentially maps to the current full deploy minus the content
 
-The output of both these builds will be deployed to a single S3 bucket with
-safeguards in place to ensure they don't override each other.
+There will be two S3 buckets. One will hold the output of the application build (including .js and .css), and the other will hold the output of the content build (including .html and .pdf).
```
Comment (billfienberg) [inline platform/engineering/design-docs/2020-04-09-separate-content-build.md:171]: Do you think it's worth clarifying that there will be three buckets total when we're done; the current bucket, the new application bucket, and the new content bucket?

Comment (bkjohnson) [reply platform/engineering/design-docs/2020-04-09-separate-content-build.md:171]: That's a good point.  Should it say:

> There will be two _new_ S3 buckets. ...
> The original bucket will still exist to hold teamsite assets.

Comment (billfienberg) [reply platform/engineering/design-docs/2020-04-09-separate-content-build.md:171]: I think that makes sense.

**Review by billfienberg — APPROVED** [review]

**Pull request merged**